Firmware/mmu2.h

```diff
@@ -30,7 +30,7 @@ struct Version {
 };
 
 /// Top-level interface between Logic and Marlin.
-/// Intentionally named MMU2 to be (almost) a drop-in replacement for the previous implementation.
+/// Intentionally named MMU to be (almost) a drop-in replacement for the previous implementation.
```

Comment:
tbh. I wouldn't change MMU2->MMU in this case :wink: - it is a comment about the MMU2 class